dht_filters.go

```diff
@@ -197,6 +197,7 @@ func isEUI(ip net.IP) bool {
 }
 
 func sameV6Net(a, b net.IP) bool {
+	//lint:ignore SA1021 We're comparing only parts of the IP address here.
```

Comment:
This is annoying, especially since we need both comments here (this new one for staticcheck, the `//nolint` for `gopls` (?)).
Furthermore, I think this is a false positive anyway: https://github.com/dominikh/go-tools/issues/957.